CHANGELOG.md

```diff
@@ -2,11 +2,7 @@
 
 ## unreleased
 
-## 3.1.0-rc.1 / 2024-12-30
-
-* [BUGFIX] OTLP receiver: Allow colons in non-standard units. #15710
-
-## 3.1.0-rc.0 / 2024-12-18
+## 3.1.0 2025-01-02
```

Comment:
I didn't add an entry for https://github.com/prometheus/prometheus/pull/15724 (thoughts it was done afterwards).
Not sure if it's worth mentioning.

Reply:
You're right; I forgot that one.